source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -164,47 +192,48 @@ The output of this code example resembles the following:
    507f1f77bcf86cd799439011 is type: org.bson.types.ObjectId
    4794261 is type: java.lang.Long
 
-To construct an instance of a ``Document`` from Extended JSON, you can call
-the ``parse()`` static factory method of the ``Document`` or ``BsonDocument``
-class. This method instantiates a ``JsonReader``, parses the Extended JSON
-string, and returns an instance of that class containing the data.
 
-The following example shows how you can use the ``Document`` class to read
-the same example Extended JSON string into a ``Document`` object using the
-``parse()`` method:
+For additional information on the classes and methods mentioned on this
+section, see the following API documentation:
 
-.. code-block:: java
+Write Extended JSON
+-------------------
 
-   String ejsonStr = "{ \"_id\": { \"$oid\": \"507f1f77bcf86cd799439011\"}," +
-                     "\"myNumber\": {\"$numberLong\": \"4794261\" }}}";
+Using the Document Classes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   Document doc = new Document(Document.parse(ejsonStr));
-   System.out.println(doc);
+You can write an Extended JSON string from an instance of ``Document`` or
+``BsonDocument`` by calling the ``toJson()`` method, passing it an instance of
```

Comment:
optionally passing

Reply:
Addressed in https://github.com/mongodb/docs-java/pull/37/commits/47fea5dd750da5601c6ba188a3bf768aa5b9a954